src/main/java/io/debezium/connector/spanner/task/SyncEventHandler.java

```diff
@@ -31,6 +34,8 @@ public class SyncEventHandler {
 
     private final BlockingConsumer<TaskStateChangeEvent> eventConsumer;
 
+    private volatile Stopwatch sw = Stopwatch.accumulating().start();
```

Comment:
Perhaps a more descriptive name would be helpful rather than just `sw`?  What is this trying to accumulate time based on for example.

Reply:
Yep, you're right, removed!